GameData/SPEngine/RealPlumeConfig.cfg

```diff
@@ -163,37 +163,7 @@
 	}
 }
 
-@PART[SPEngineGamma]:FOR[ZZREALPLUME]:NEEDS[SmokeScreen]
-{
-	PLUME
-	{
-		name = HTP_RP1_Lower
-		transformName = thrustTransform
-		localRotation = 0,0,0
-		localPosition = 0,0,0
-		energy = 1
-		speed = 1
-		flarePosition = 0,0,0
-		flareScale = 0.1
-		plumePosition = 0,0,0
-		plumeScale = 0.5
-		fumePosition = 0,0,0.1
-		fumeScale = 0.5
-	}
-	@MODULE[ModuleEngines*]
-	{
-		%powerEffectName = HTP_RP1_Lower
-	}
-	@MODULE[ModuleEngineConfigs]
-	{
-		@CONFIG,*
-		{
-			%powerEffectName = HTP_RP1_Lower
-		}
-	}
-}
-
-@PART[SPEngineGamma2]:FOR[ZZREALPLUME]:NEEDS[SmokeScreen]
+@PART[SPEngineGamma|SPEngineGamma2|SPEngineGammaAlt]:FOR[ZZREALPLUME]:NEEDS[SmokeScreen]
```

Comment:
If there is one, SPEngineGamma2 (W-class) should use an HTP_RP1_Upper plume, as it's an upper stage with a long vacuum nozzle.

Reply:
let me have a look

Reply:
Realplume doesn't have a upper stage htp rp1 plume

Reply:
:cry: okay, this'll have to do then.

I'll try to give this PR a test later today, and merge if all is good.

Reply:
no worrys